proposed/longhorn/latest/templates/deployment-ui.yaml

```diff
@@ -14,6 +14,9 @@ spec:
     metadata:
       labels:
         app: longhorn-ui
+        {{- if eq .Values.service.ui.type "Rancher-Proxy" }}
+        kubernetes.io/cluster-service: "true"
+        {{- end }}
```

Comment:
the `kubernetes.io/cluster-service: "true" ` should be added to the service labels instead of the deployment labels.